lib/room.rb

```diff
@@ -1,12 +1,21 @@
-class Rom
-
+class Room
+attr_reader :category
   def initialize(category, length, width)
     @category = category
-    @length = 10
+    @length = length
     @width = width
+    @paint = false
   end
 
   def area
-    @length * @width
+    @length * @width.to_i
+  end
+
+  def is_it_painted?
```

Comment:
Make sure to name methods exactly what they are named in the README/spec. You will lose points because if you do not because our tests expect them to be named exactly the way they are written in the README.